docs/decisions/0002-footer-customizations.rst

```diff
@@ -0,0 +1,138 @@
+to create and maintain forks of the repositories.
+
+With the existing mechanisms for customizing the MFEs it is not possible to
+change the content or layout of selected components without creating a fork of
+a desired MFE and making the change directly in the code, which requires a lot
+of effort to maintain.
+
+It is important to reduce the gap in customisability between the
+legacy UI and MFEs, while keeping the maintenance costs for the customisation
+options low. Otherwise, the vendors and their clients would have an unpleasant
+decision to make - either to forgo any advanced or niche UI customisations, or
+pay a high recurring cost for maintaining them.
+
+Coming up with a mechanism for the MFEs that would allow the same degree of
+customization would be a big undertaking and is outside of the scope of this
+ADR. Instead, the suggested approach for the time being is to provide more
+customisation options which would cover a wide number of cases, so that forking
+a repository would be needed less frequently. Thus, the scope of this ADR is
+defining conventions for how customizations should be exposed through the
+existing APIs.
+
+Footer is a component used in every (or almost every) MFE. The mechanisms
+for customizing it should make it easy to set the options once for all the
+MFEs that use it, while still being flexible.
+
+.. _legacy UI theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/index.html
+
+Decision
+********
+
+.. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+There are to two types of customizations - those that alter the styles and
+those that alter the content of the component. We will expose both
+customization option to be configurable through the `MFE's config mechanism`_.
+
+For styles, to allow a great degree of freedom in customization, we will allow,
+where appropriate, to either supply a list of CSS classes which will be applied
+to a particular component, or a list of CSS attributes and values.
+
+.. _MFE's config mechanism: https://github.com/openedx/frontend-platform/blob/master/src/config.js
+
+Consequences
+************
+
+.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+Positive
+========
+
+* Styles and additional content (i.e. all the customizations to the component)
+  can be configured in the same place (or at least using the same API).
+* Can set defaults at build time, and overwrite them at runtime.
+* Have the option to do at build time.
+* Can customize for each MFE independently.
+* Using ``eox-tenant`` can customize per tenant (org, microsite).
```

Comment:
I think we can leave out eox-tenant here since it's a plugin that isn't installed by default.